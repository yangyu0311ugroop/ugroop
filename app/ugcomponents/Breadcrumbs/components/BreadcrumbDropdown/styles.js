const styles = ({ colors, fontSize }) => ({
  btn: {
    padding: 0,
    margin: 0,
    minWidth: 'auto',
    '&:hover': {
      backgroundColor: 'transparent',
    },
  },
  link: {
    padding: '16px 32px',
    display: 'block',
    fontWeight: '600',
    color: colors.breadcrumb.normal,
    fontSize: fontSize.breadcrumb.item,
    '&:hover': {
      color: colors.breadcrumb.normal,
      textDecoration: 'none',
    },
    '&:visited': {
      colors: colors.breadcrumb.normal,
    },
  },
  icon: {
    marginLeft: '16px',
    color: colors.breadcrumb.chevron,
    fontSize: '15px',
  },
});

export default styles;
