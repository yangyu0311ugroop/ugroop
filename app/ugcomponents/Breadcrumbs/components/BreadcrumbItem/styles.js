const styles = ({ colors, fontSize }) => ({
  stackItem: {
    padding: '16px 32px',
    display: 'block',
  },
  stackContainer: {
    padding: 16,
  },
  listItem: {
    display: 'flex',
    alignItems: 'center',
    marginLeft: '8px',
    '&:first-child': {
      marginLeft: '0',
    },
  },
  linkText: {
    fontWeight: 'normal',
    color: colors.breadcrumb.normal,
    fontSize: fontSize.breadcrumb.item,
    textDecoration: 'none',
    '&:hover': {
      color: colors.breadcrumb.normal,
      textDecoration: 'underline',
    },
    '&:visited': {
      colors: colors.breadcrumb.normal,
    },
  },
  chev: {
    marginLeft: '8px',
    color: colors.breadcrumb.chevron,
    fontSize: '15px',
  },
  homeActive: {
    fontWeight: 'normal',
    color: colors.breadcrumb.active,
    textDecoration: 'none',
    '&:hover': {
      color: colors.breadcrumb.active,
      textDecoration: 'underline',
    },
    '&:visited': {
      colors: colors.breadcrumb.active,
    },
  },
  home: {
    fontSize: fontSize.breadcrumb.item,
  },
  darkMode: {
    color: 'white',
    textDecoration: 'none',
    '&:hover': {
      color: 'white',
      textDecoration: 'underline',
    },
    '&:visited': {
      color: 'white',
    },
  },
});

export default styles;
