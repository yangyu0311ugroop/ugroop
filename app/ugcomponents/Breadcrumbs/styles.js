const stylesheet = ({ colors }) => ({
  root: {
    display: 'flex',
    alignItems: 'center',
  },
  showMore: {
    padding: 0,
    margin: 0,
    minWidth: 'auto',
  },
  chev: {
    marginLeft: '16px',
    color: colors.breadcrumb.chevron,
    fontSize: '12px',
  },
  popoverContainer: {
    padding: 16,
  },
  darkMode: {
    color: 'white',
  },
});

export default stylesheet;
