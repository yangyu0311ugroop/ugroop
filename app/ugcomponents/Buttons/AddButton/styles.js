const styles = {
  root: {
    border: 'unset',
    background: 'unset',
    color: '#333',
    paddingLeft: 3,
    fontSize: 12,
  },
  grow: {
    flex: '1',
  },
  smallText: {
    fontSize: 12,
    padding: '2px 8px',
    minHeight: 'unset',
    fontWeight: 500,
  },

  addIcon: {
    background: '#39bd3f',
    boxShadow: '0 1px 0 0 #1a981f',
    color: 'white',
    borderRadius: '50%',
    paddingLeft: 5,

    width: 21,
    height: 21,
  },
};

export default styles;
