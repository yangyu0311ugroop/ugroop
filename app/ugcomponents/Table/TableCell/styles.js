const styles = {
  root: {
    fontSize: '14px',
    '&:active': {
      color: '#2b344d',
    },
  },
  noPadding: {
    padding: '0',
    '&:last-child': {
      paddingRight: '0',
      textAlign: 'right',
    },
  },
};

export default styles;
