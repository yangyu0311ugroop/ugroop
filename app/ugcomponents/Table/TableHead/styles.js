const styles = {
  root: {
    // Add some styles here
  },
};

export default styles;
