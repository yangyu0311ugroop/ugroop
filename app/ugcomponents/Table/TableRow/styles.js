const styles = {
  root: {
    // Add styles here
  },
};

export default styles;
