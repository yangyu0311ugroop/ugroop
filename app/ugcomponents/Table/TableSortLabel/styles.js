const styles = {
  root: {},
  icon: {
    display: 'none',
  },
  ownIcon: {
    marginLeft: '8px',
  },
};

export default styles;
