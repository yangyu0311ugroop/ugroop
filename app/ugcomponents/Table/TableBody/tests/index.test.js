import { shallow } from 'enzyme';
import toJSON from 'enzyme-to-json';
import React from 'react';
import mockStylesheet from 'utils/mockStylesheet';
import { TableBody } from '../index';
import styles from '../styles';

const mockStyle = mockStylesheet('TableBody', styles);

describe('<TableBody />', () => {
  let rendered;

  const props = {
    classes: mockStyle,
  };

  beforeEach(() => {
    rendered = shallow(<TableBody {...props} />);
  });

  it('should exists', () => {
    expect(TableBody).toBeDefined();
  });

  it('should render without exploding', () => {
    expect(rendered.length).toBe(1);
  });

  it('should render something', () => {
    expect(toJSON(rendered)).toMatchSnapshot();
  });
});
