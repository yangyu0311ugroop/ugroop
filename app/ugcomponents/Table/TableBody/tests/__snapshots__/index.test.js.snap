// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<TableBody /> should render something 1`] = `
<WithStyles(ForwardRef(TableBody))
  className="TableBody-root"
/>
`;
